components/hmpo-cookie-banner/_style.scss

```diff
@@ -1,13 +1,31 @@
 
 .hmpo-cookie-banner {
-    @include govuk-font($size: 16);
+    @include govuk-font($size: 19, $line-height: 1.25);
     @include govuk-text-colour;
 
-    width: 100%;
+    background-color: govuk-colour("light-grey");
+    margin: 0;
+    padding: govuk-spacing(4) 0;
+}
+
+.hmpo-cookie-banner__button-wrapper {
+    padding: 0;
+}
+
+.hmpo-cookie-banner__text {
+    margin: 0 0 govuk-spacing(2);
+}
+
+.hmpo-cookie-banner__button {
+    @include mq($until: desktop) {
+        width: 100%;
+        margin: 0 0 govuk-spacing(4) 0;
+    }
```

Comment:
have the mobile/tablet (mobile first) as the default css for this class, i.e. not inside a media query,
then have the desktop and bigger in a media query.